main.tf

```diff
@@ -263,6 +275,7 @@ resource "aws_s3_bucket_acl" "replication" {
 
 # Configure bucket lifecycle rules
 resource "aws_s3_bucket_lifecycle_configuration" "replication" {
+  #checkov:skip=CKV_AWS_300: "Ensure S3 lifecycle configuration sets period for aborting failed uploads"
```

Comment:
See if you can address this issue, rather than suppressing it.
If there is a reason why you cannot implement it add an explanation why it is being ignored.

Reply:
We have decided to do this under a new request so it will not be covered here